src/misc/setup.cpp

```diff
@@ -1428,21 +1464,23 @@ void SETUP_ParseConfigFiles(const std::string &config_path)
 	std::string config_file;
 
 	// First: parse the user's primary config file
-	const bool wants_primary_conf = !control->cmdline->FindExist("-noprimaryconf", true);
+	const bool wants_primary_conf =
+	        !control->cmdline->FindExistOption("noprimaryconf", true);
 	if (wants_primary_conf) {
 		Cross::GetPlatformConfigName(config_file);
 		const std::string config_combined = config_path + config_file;
 		control->ParseConfigFile("primary", config_combined);
 	}
 
 	// Second: parse the local 'dosbox.conf', if present
-	const bool wants_local_conf = !control->cmdline->FindExist("-nolocalconf", true);
+	const bool wants_local_conf =
+	        !control->cmdline->FindExist("nolocalconf", true);
 	if (wants_local_conf) {
 		control->ParseConfigFile("local", "dosbox.conf");
 	}
 
 	// Finally: layer on custom -conf <files>
-	while (control->cmdline->FindString("-conf", config_file, true)) {
+	while (control->cmdline->FindString("conf", config_file, true)) {
```

Comment:
When I read these three uses of `cmdline->Find..`, I'm not sure what the difference is between:
 - `cmdline->FindExistOption(...)`, versus
 - cmdline->FindExist(...)`

The `Exist` adjustective is confusing here.

Is it possible to find non-existing options? If we found something, then it's implied they already exist?  

Maybe we can call your new function(s) simply: `FindOption()`?

Reply:
I previously tried to make it match the naming of functions like FindStringOption and FindIntOption, but perhaps this is not necessary and may actually cause confusion. Renamed FromExistOption to FindOption as suggested.